common/djangoapps/dark_lang/tests.py

```diff
@@ -42,33 +43,70 @@ def setUp(self):
             enabled=True
         ).save()
 
-    def process_request(self, language_session_key=UNSET, accept=UNSET, preview_lang=UNSET, clear_lang=UNSET):
+    def process_post(self, language_session_key=UNSET, accept=UNSET, preview_lang=UNSET, clear_lang=UNSET):
         """
-        Build a request and then process it using the ``DarkLangMiddleware``.
+        Build the POST request and set the language settings according to parameters
```

Comment:
Similarly, update the docstring to make it clear that it's building the POST request and then actually processing the request.